### PR TITLE
Feat/tablet loyout

### PR DESCRIPTION
**Pull request by AlonaLook**
Please, review and merge layout for view on a tablet.

### CODE REVIEW

File: src/css/modules/cards.scss [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+      margin-bottom: 0;
+    }
+
+    @media screen and (min-width: $break-tablet) {
+      margin-bottom: 26px;
+    }
+  }
+
+  &__title {
+    color: #000;
```
Comment (PritulaRoman) [inline src/css/modules/cards.scss]: Please, use colors from `variables`

File: src/css/modules/buttons.scss [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  &__item {
+    height: 48px;
+    box-sizing: border-box;
+    text-transform: uppercase;
+    background: $primary-color;
+    border-radius: 4px;
+    font-size: 12px;
+    font-weight: bold;
+    padding: 15px 24px;
+    border: 2px solid rgba(255, 255, 255, 0.7);
```
Comment (PritulaRoman) [inline src/css/modules/buttons.scss]: Please, use colors from variables

File: src/css/modules/cards.scss (hunk 1 above)
Comment (AlonaLook) [reply src/css/modules/cards.scss]: Changed.

File: src/css/modules/buttons.scss (hunk 2 above)
Comment (AlonaLook) [reply src/css/modules/buttons.scss]: Changed.

**Pull request merged**